article.tex

```diff
@@ -1419,6 +1419,115 @@ \subsection*{Table 5 - SMARTS matching performance.}\label{tab:smigrep}
 \end{tabular}
 
 
+\subsection*{Table ? }
+\begin{tabular}{llllllll}
+Benchmark  & Data Set & Input Format & \multicolumn{2}{c}{CDK 1.4.19} & \multicolumn{2}{c}{CDK 2.0} & Improvement \\
```

Comment:
Sure, I use the macro originally because we hadn't decided whether it would be 1.6 or 2.0 :-)